learnArrays/learnArrays.js

```diff
@@ -0,0 +1,58 @@
+// function to find maximum number in the array
+// Args : array of numbers
+
+export let findMax = (testArr) => { // [2,3,4]
```

Comment:
Why did you use `let` here? why not `const` ? Same applies for other functions as well.

Reply:
fixed